var/spack/repos/builtin/packages/py-neurodamus/package.py

```diff
@@ -33,6 +33,7 @@ class PyNeurodamus(PythonPackage):
     # We depend on Neurodamus but let the user decide which one
     depends_on('python@3.4:',      type=('build', 'run'))
     depends_on('py-setuptools',    type=('build', 'run'))
+    depends_on('neuron',           type='run')
```

Comment:
Thinking about it, maybe `neuron` was not loaded intentionally, and the previous comment should be changed to
> _We depend on NEURON but let the user decide which one_

?

Reply:
@ferdonline is right person for this to answer.

(on BB5 deployment we typically have a single neuron installation and hence adding neuron here is not harmless)

Reply:
I'll leave this for @ferdonline , from my side the rest of this PR is good.

Reply:
I answered question about this on HPC slack : due to issues with Intel and GCC conflicts, we don't suggest to add this dependency.

Reply:
> due to issues with Intel and GCC conflicts, we don't suggest to add this dependency

Can this impact us when people load both packages?  ie: will we be getting an incompatible mixture of intel & gcc compiled code?